ci/docker/runtime_functions.sh

```diff
@@ -411,8 +413,9 @@ build_ubuntu_cpu_mkl() {
         -DUSE_MKLDNN=OFF \
         -DUSE_CUDA=OFF \
         -DUSE_TVM_OP=ON \
-        -DUSE_MKL_IF_AVAILABLE=ON \
+        -DUSE_MKL_IF_AVAILABLE=OFF \
```

Comment:
why are we switching this off? This test is specifically for MKL as BLAS engine. Removing this would cause all the subsequent PRs to not check for MKL as BLAS engine.